coordination/src/main/java/tech/ydb/coordination/impl/ClientImpl.java

```diff
@@ -39,6 +40,12 @@ private String validatePath(String path) {
         return path.startsWith("/") ? path : rpc.getDatabase() + "/" + path;
     }
 
+    private GrpcRequestSettings makeGrpcRequestSettings(BaseRequestSettings settings) {
```

Comment:
static method?

Reply:
It's not necessary. Every client creates the grpc settings in its own way. It can not have the state right now but why it can have it in the future? For example the table session implementation have it